src/request.rs

```diff
@@ -32,7 +33,7 @@ pub trait KvRequest: Sync + Send + 'static + Sized {
 
     fn execute(self, pd_client: Arc<impl PdClient>) -> BoxFuture<'static, Result<Self::Result>> {
         Self::reduce(
-            self.response_stream(pd_client)
+            self.response_stream(pd_client, LOCK_RETRY_MAX_TIMES)
```

Comment:
I'm thinking if we can combine `LOCK_RETRY_MAX_TIMES` into backoff because backoff has the function of retry limit. Maybe give PessimisticLock request a custom backoff?